tools/scripts/release_package.py

```diff
@@ -91,6 +91,34 @@ def _update_android_versions(version: str, github_token: str):
 
     uv._update_android_version(version)
 
+def _update_native_sdk_versions(version: str, arg_ios_version:str, arg_android_version: str, native_sdk_versions_path: str):
+    ios_version = arg_ios_version or _get_current_ios_version()
+    android_version = arg_android_version or _get_current_android_version()
```

Comment:
The native sdk versions are either obtained as a provided parameter or directly read from DatadogDependencies.xml if not provided.